fiftyone/server/routes/dynamic_group.py

```
@@ -0,0 +1,239 @@
+    """
+    header = request.headers.get("If-Match")
+    if not header:
+        return None
+
+    value, _ = utils.http.ETag.parse(header)
+
+    raw = value
+    try:
+        raw = base64.b64decode(value.encode("utf-8")).decode("utf-8")
+    except Exception:
+        pass
```

Ruff (0.16.1)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)